### PR TITLE
refactor(config): deprecate apiEndpoint in favor of jsSrc 

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #91

### CODE REVIEW

**Comment by DSergiu** [comment]
Missing changelog update and version bump. Looks good to go. @CAMOBAP

File: sdk/build.gradle [hunk 1]
```diff
@@ -22,11 +22,11 @@ android {
         // See https://developer.android.com/studio/publish/versioning
         // versionCode must be integer and be incremented by one for every new update
         // android system uses this to prevent downgrades
-        versionCode 26
+        versionCode 27
 
         // version number visible to the user
         // should follow semantic versioning (See https://semver.org)
-        versionName "3.4.0"
+        versionName "3.4.1"
```
Comment (DSergiu) [inline sdk/build.gradle:29]:
```suggestion
        versionName "3.5.0"
```

Comment (DSergiu) [reply sdk/build.gradle:29]: Should be a minor update (according to semver) since this is no bug fix.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 3.4.1
```
Comment (DSergiu) [inline CHANGES.md:3]:
```suggestion
# 3.5.0
```

**Comment by CAMOBAP** [comment]
Checking `Unknown platform error occurred when running the UTP test suite` which permanently occurs for UI Tests

**Review by DSergiu — APPROVED** [review]

**Pull request merged**